native/src/main/scala/sqs4s/internal/models.scala

```diff
@@ -0,0 +1,177 @@
+package sqs4s.internal
+
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets
+import java.time._
+import java.util.concurrent.TimeUnit
+
+import cats.effect.{Clock, Sync}
+import cats.implicits._
+import org.http4s.Credentials.Token
+import org.http4s.headers.Authorization
+import org.http4s.util.CaseInsensitiveString
+import org.http4s.{Headers, Method, Query, Request, Uri}
+
+import scala.language.postfixOps
+
+private[sqs4s] object models {
+  import aws4.common._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)